bats/tests/helpers/vm.bash

```diff
@@ -81,11 +81,11 @@ setup_ramdisk() {
 
     local ramdisk_size="${RD_RAMDISK_SIZE}"
     if ((ramdisk_size < ${RD_FILE_RAMDISK_SIZE:-0})); then
-        run printf "%s requires %dGB of ramdisk; disabling ramdisk for this file" \
-            "$BATS_TEST_FILENAME" "$RD_FILE_RAMDISK_SIZE"
-        assert_success
-        printf "RD:   %s\n" "$output" >>"$BATS_WARNING_FILE"
-        printf "# WARN: %s\n" "$output" >&3
+        local fmt='%s requires %dGB of ramdisk; disabling ramdisk for this file'
+        # shellcheck disable=SC2059 # The string is set the line above.
+        fmt="$(printf "$fmt" "$BATS_TEST_FILENAME" "$RD_FILE_RAMDISK_SIZE")"
```

Comment:
I learned yesterday that `printf` is a bash built-in that could be used like this:

```suggestion
        printf -v fmt "$fmt" "$BATS_TEST_FILENAME" "$RD_FILE_RAMDISK_SIZE"
```